### ISSUE TITLE
Add `jemalloc` dependency to README

### ISSUE THREAD

**Issue by IronGauntlets** [NethermindEth/juno#1724]

**Comment by aminsato** [comment]
Do you mean how to install Jemalloc dependencies, like `brew insall pkg-config`?

**Comment by PhilippeR26** [comment]
It was straightforward to build juno from source in the previous versions.
But seems complicated to build 0.9.4.
I have :
```
> brew list
binutils  gcc  gmp  isl  jemalloc  libmpc  lz4	mpfr  pkg-config  xz  zlib  zstd
```
`make juno` works without error, but when launching :
```
./build/juno: error while loading shared libraries: libjemalloc.so.2: cannot open shared object file: No such file or directory
```

Could be useful to have a more detailed `Build and Run` chapter in the README file, made for a fresh computer.

Temporarily, I came back to 0.9.3 build.

**Comment by stefanalfbo** [comment]
I made a PR on the extra steps I needed to do to be able to build the project.

**Comment by PhilippeR26** [comment]
Nice.
These instructions are working for me.

**Comment by IronGauntlets** [comment]
closed by #1739
